applications/ironside/se/update/Kconfig

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config UPDATE_BLOB_ADDRESS
+	hex
+	default 0x0e100000
+	help
+	  Address to the update blob
```

Comment:
any reason this can't come from dts?

Reply:
This is a property of the zip file containing the upgrade, not the device itself. My understanding is that dts should be used to describe the device, while this is not related to the device itself.